class-toc-plus.php

```diff
@@ -325,23 +325,45 @@ public function shortcode_sitemap( $atts ) {
 
 
 		public function shortcode_sitemap_pages( $attributes ) {
+			global $post;
+			$args_child_of = 0;
```

Comment:
I think you can avoid using a new local var to store this. Just use `$atts['child_of']` where you can.